### DISCUSSION TITLE
ScriptReference/GizmoUtility

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#667] [Notes]
# ScriptReference/GizmoUtility

https://docs.unity3d.com/ScriptReference/GizmoUtility.html

**Comment by andreasIsCoding** [comment]
These methods can be accessed in Unity versions prior to 2022 in the internal class "AnnotationUtility" with reflection:
https://discussions.unity.com/t/find-current-scale-of-3d-gizmos/44310
